### PR TITLE
WRF 4.4 for ARCHER2

### PR DESCRIPTION
**Pull request by kevinstratford**
Update module commands, and update to v4.4.
Some minor updates.

### CODE REVIEW

File: apps/WRF/ARCHER2-WRF-4.4/build-jasper-cray.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# From the default envoronment
```
Comment (juanfrh) [inline apps/WRF/ARCHER2-WRF-4.4/build-jasper-cray.sh:3]: _Environment_ is misspelled.

File: apps/WRF/ARCHER2-WRF-4.4/build-libpng-gnu.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# From the default envoronment
```
Comment (juanfrh) [inline apps/WRF/ARCHER2-WRF-4.4/build-libpng-gnu.sh:3]: _Environment_ is misspelled.

File: apps/WRF/ARCHER2-WRF-4.4/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Then WRF and WPS can be built
+
+```
+$ bash ./build-wrf-gnu.sh
+$ bash ./build-wps-gnu.sh
+```
+
+Allow 15-20 minutes for this build.
+
+The module environemt at the time of writing was:
```
Comment (juanfrh) [inline apps/WRF/ARCHER2-WRF-4.4/README.md:61]: _Environment_ is misspelled.

**Review by juanfrh — COMMENTED** [review]
Just a few typos. Otherwise, LGTM.

File: apps/WRF/ARCHER2-WRF-4.4/build-jasper-cray.sh (hunk 1 above)
Comment (kevinstratford) [reply apps/WRF/ARCHER2-WRF-4.4/build-jasper-cray.sh]: Oh dear. Can't spell...

**Pull request merged**